docs/agile-process.md

```diff
@@ -77,15 +80,15 @@ All teams members to provide update in github issues in progress on Monday/early
 
 ### Code Review (In Progress->Ready For Review->Done)
 
-Pull requests are reviewed on a per-ticket basis for merge into test. Better to push to test more rather than less; we assume test will break.
-
-Stories are reviewed against acceptance criteria (AC) with test coverage.
+Pull requests are reviewed on a per-ticket basis and tests are automatically run on the CI server. One reviewer required to merge any pull request and anyone can review, relecting our preference for velocity. Signal to the team in slack if you've waited more than two hours for a review.
```

Comment:
What do you mean by "relecting our preference for velocity"? First, should that be "reflecting"? Second, I'm wondering what this preference is over. Like I'd rather spend a few revisions making more accurate/reliable code than optimize to get it out the door faster.

Reply:
@geekygirlsarah Thanks for catching the typo, fixed. It's been a week or two since we discussed pull request process, but my recollection (I could be wrong) is that the velocity comment in the context of code review came from @tadhg-ohiggins. I defer to engineering for how to describe what seems to be a shared preference towards merging the PR rather than waiting.

Reply:
I don't recall making that comment about velocity; I think changing it to:
> Approval from one reviewer is required to merge a pull request. Reviewer is welcome to merge but responsibility to merge is with the submitter.
would work.